data/root/config

```diff
@@ -1,4 +1,4 @@
-root:			common root cracklib-dict-full.rpm bind ?yast2-trans-<lang>.rpm
+root:			common root libstoragemgmt cracklib-dict-full.rpm bind ?yast2-trans-<lang>.rpm
```

Comment:
OK, but then I'm perhaps missing something about cracklib-dict-full.rpm - it is loaded later by YaST...